bin/peanut.dart

```diff
@@ -12,6 +12,7 @@ void main(List<String> args) {
   ArgParser parser = new ArgParser(allowTrailingOptions: false)
     ..addOption(_directoryFlag, abbr: 'd', defaultsTo: 'web')
     ..addOption('branch', abbr: 'b', defaultsTo: 'gh-pages')
+    ..addOption('mode', defaultsTo: 'release')
```

Comment:
Update this to include only the valid options: I think it's `release` and `debug`? Or dev? Whatever it is.